src/rgw/rgw_user.cc

```diff
@@ -1735,6 +1735,7 @@ int RGWUser::init(RGWUserAdminOpState& op_state)
   }
 
   if (!uid.empty() && (uid.compare(RGW_USER_ANON_ID) != 0)) {
+    op_state.set_reserved_user(false);
```

Comment:
@mikulely Then the `reserved_user` should be `false` right ? I understand that `anonymous user` and `reserved_user` are different.

>     if (!uid.empty() && (uid.compare(RGW_USER_ANON_ID) != 0)) { 
>         op_state.set_reserved_user(false); 

^^ So this is the place the reserved_user becomes true, right ?